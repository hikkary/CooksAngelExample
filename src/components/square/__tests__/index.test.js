
import React from 'react';
import Square from '../';
import 'jest-styled-components';

import { shallow } from 'enzyme';


it('should match the snapshot (good Practice)', () => {
  const component = shallow(
    <Square  />
  ).dive();
  // console.log(component)
  // console.log(component.debug())
  // console.log(component.props())
  // console.log(component.state())
  expect(component).toMatchSnapshot();
});

