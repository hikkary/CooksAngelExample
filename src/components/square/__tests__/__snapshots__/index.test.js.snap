// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot (good Practice) 1`] = `
<PoseElement
  elementType={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  margin: 20px;
  height: 50px;
  width: 50px;
  background-color: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  innerRef={null}
  poseConfig={
    Object {
      "end": Object {
        "transition": Object {
          "duration": 1000,
        },
        "x": 100,
      },
      "start": Object {
        "transition": Object {
          "duration": 1000,
        },
        "x": 0,
      },
    }
  }
/>
`;
